doc/development_guide/how_tos/custom_checkers.rst

```diff
@@ -113,7 +113,13 @@ It has the following format::
 * The ``option-symbol`` is a unique name for the option.
   This is used on the command line and in config files.
   The hyphen is replaced by an underscore when used in the checker,
-  similarly to how you would use  ``argparse.Namespace``.
+  similarly to how you would use  ``argparse.Namespace``:
+
+  .. code-block:: python
+
+    if self.linter.config.ignore_ints:
+        # skip check
+        pass
```

Comment:
```suggestion
        return

Reply:
That was my original proposal, but `rstcheck` failed in CI:
`doc/development_guide/how_tos/custom_checkers.rst:122: (ERROR/3) (python) 'return' outside function`

I didn't want to make the code sample too big

Reply:
Maybe inverting the conditional and using an ellipsis instead of the pass then?

Reply:
Yep, that works